.travis.yml

```diff
@@ -1,25 +1,28 @@
 dist: trusty
 sudo: false
 
-install:
-  - make install-devtools
-  - make deps
-
 language: go
 
 go:
-  # - 1.7.x (abandonned - not supported by dep)
-  - "1.8.x"
-  - "1.9.x"
-  - "1.10.x"
+  #- "1.8.x"
+  #- "1.9.x"
+  #- "1.10.x"
   - "1.11.x"
-  - tip
+  #- tip
```

Comment:
did you have to remove all the older versions of `go` on purpose? are there limitations with regards to golang versions on Windows builders on Travis-ci ?

Reply:
I removed the old versions to keep from hammering Travis while I tested.  I'll put them back before I'm done here.